node/src/test/scala/coop/rchain/node/mergeablity/JoinMergeability.scala

```diff
@@ -85,9 +84,8 @@ class JoinMergeability extends FlatSpec with Matchers with Inspectors with Merge
     R0.rstate ++ S1.rstate
   )
   it should "R J 4" in MergeableCase(J_)(F1)(R0)(J_.rstate ++ R0.rstate ++ F1.rstate)
-  it should "R J 4 2" in ConflictingCase(J_)(F0)(R0)(J_.rstate ++ R0.rstate)(R0.rstate)
-  it should "R J C" in MergeableCase(C0)(J_)(R0)(J_.rstate ++ R0.rstate)
-  it should "R J C 2" in MergeableCase(C1)(J_)(R0)(J_.rstate ++ R0.rstate ++ C1.rstate)
+  it should "R J C" ignore MergeableCase(C0)(J_)(R0)(J_.rstate ++ R0.rstate)                // infinite loop
+  it should "R J C 2" ignore MergeableCase(C1)(J_)(R0)(J_.rstate ++ R0.rstate ++ C1.rstate) // infinite loop
   it should "R J R" in ConflictingCase(R0)(J_)(R0)(R0.rstate ++ R0.rstate)(
```

Comment:
I'm not sure - were these tests working before? There is nothing changed related to test execution, right?

Reply:
The codes didn't check if they are conflict or not. The codes only check the state before this pr. see line here https://github.com/rchain/rchain/pull/3430#discussion_r667345172

Reply:
Let me check again. The commits are too old in my memory.

Reply:
Ok. I think you are right on this one. I remember why I ignore the test case. It is because I put a more strict rule for the test case -> https://github.com/rchain/rchain/blob/33d8a16d552762be05c973634521063144ba86fc/node/src/test/scala/coop/rchain/node/mergeablity/ComputeMerge.scala#L79-L83

when the deploy failed with error and raise the error directly. In this case, the deploy run into infinite loop which the deploy would fail with run out of phlo. And I notice that all other old InfiniteLoop tests are commented out -> https://github.com/rchain/rchain/blob/9cfb8954d5f44ccf8940ddb473268e9844e470b6/casper/src/test/scala/coop/rchain/casper/batch1/BaseMergeability.scala#L20-L22
 so I just ignore these tests.

Reply:
I see, ok thanks for explanation